client/app/cloud/project/compute/volume/snapshot/cloud-project-compute-volume-snapshot-add.controller.js

```diff
@@ -20,13 +20,11 @@ angular.module("managerApp")
 
         function init () {
             self.loaders.init = true;
-            OvhApiCloudPrice.Lexi().query().$promise.then(function (prices) {
-                var price = _.find(prices.snapshots, function (price) {
-                    return price.region === self.snapshot.volume.region;
-                });
-                if (price && price.monthlyPrice) {
-                    self.snapshot.price = price.monthlyPrice.value * self.snapshot.volume.size;
-                    self.snapshot.priceText = price.monthlyPrice.text;
+            OvhCloudPriceHelper.getPrices(serviceName).then(function (prices) {
+                var price = prices[`volume.snapshot.consumption.${self.snapshot.volume.region}`] || prices[`volume.snapshot.consumption`];
```

Comment:
this looks sketchy... Maybe use some variable?
const volumeSnapshotConsumption?